packages/daimo-common/src/daimoLink.ts

```diff
@@ -6,6 +6,10 @@ import { BigIntStr, DollarStr, zDollarStr, zHex } from "./model";
 export const daimoDomain =
   process.env.NEXT_PUBLIC_DOMAIN || process.env.DAIMO_DOMAIN;
 
+export const daimoLinkBaseFuture = daimoDomain
+  ? `https://${daimoDomain}/l`
+  : "http://localhost:3001/l";
```

Comment:
how about `daimoLinkV1` and `daimoLinkV2`

Reply:
or just `daimoLinkBase` with the new form > hardcode in `daimo.com/link` the same way we hardcode `daimo.xyz/link` backcompat below

Reply:
`daimo.xyz/link` backcompat has actually been incomplete, it doesn't work for the web deeplinks (have to export a variable for that like above). Renamed to `daimoLinkBaseV2`